automatic/jenkins/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,64 @@
+$ErrorActionPreference = 'Stop'
+
+$ToolsDir = Split-Path -parent $MyInvocation.MyCommand.Definition
+Import-Module $ToolsDir\helpers.psm1
+
+$PackageArgs = @{
+    packageName   = $env:ChocolateyPackageName
+    fileType      = 'msi'
+    file          = Join-Path $ToolsDir 'jenkins.msi'
```

Comment:
I mustt have missed this in the initial review, can you change this one to follow the same conventions we already have in the repository:

```suggestion
    file          = "$ToolsDir\jenkins.msi"
```

Then do a replacement of the filename in the `update.ps1` script (in case it changes in the future). See the calibre package as an example: https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/calibre/update.ps1#L22-L24 (variable name is different, but that is fine).